docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -180,7 +180,7 @@ h|Kafka property name
 |===
 
 ifdef::community[]
-NOTE: If you try to create a new topic with a number of partitions that would cause the partition limit of the Kafka instance to be exceeded, you see an error message indicating this. If you try to increase the number of partitions for an existing topic and this would cause the cause the partition limit of the Kafka instance to be exceeded, you see an error message stating that topic authorization failed.
+NOTE: If you try to create a new topic with a number of partitions that would cause the partition limit of the Kafka instance to be exceeded, you see an error message indicating this. If you try to increase the number of partitions for an existing topic this would cause the partition limit of the Kafka instance to be exceeded and you would see an error message stating that topic authorization failed.
```

Comment:
Good catch, @fmcdonal. However, the new version changes the meaning in a way that's not quite correct.

We just need to delete the duplicate "cause the" and maybe add the word "increase" for clarity, resulting in this:

```
NOTE: If you try to create a new topic with a number of partitions that would cause the partition limit of the Kafka instance to be exceeded, you see an error message indicating this. If you try to increase the number of partitions for an existing topic and this increase would cause the partition limit of the Kafka instance to be exceeded, you see an error message stating that topic authorization failed.
```